pkg/util/utils_windows.go

```diff
@@ -0,0 +1,8 @@
+// +build windows
+
+package util
+
+// GetRootlessRuntimeDir returns the runtime directory when running as non root
+func GetRootlessRuntimeDir() (string, error) {
+	return "", nil
```

Comment:
I'd throw an ErrNotImplemented or similar here

Reply:
Using the libpod Errors results in an import circular dep, i did an errors.New here.

Reply:
Works for me